target/linux/realtek/rtl839x/config-5.10

```diff
@@ -12,7 +12,8 @@ CONFIG_CLKDEV_LOOKUP=y
 CONFIG_CLKSRC_MMIO=y
 CONFIG_CLONE_BACKWARDS=y
 CONFIG_COMMON_CLK=y
-CONFIG_COMMON_CLK_BOSTON=y
+# CONFIG_COMMON_CLK_BOSTON is not set
```

Comment:
probably very valid; probably should be in its own cosmetic comment :(

Reply:
dropped